code/game/gamemodes/miniantags/borer/borer_event.dm

```diff
@@ -1,7 +1,7 @@
 /datum/round_event_control/borer
 	name = "Borer"
 	typepath = /datum/round_event/borer
-	weight = 10 //Default weight
+	weight = 0 //Default weight
 	max_occurrences = 1
```

Comment:
Could probably change this to 0, as well, so the event system doesn't fuck up during a long round and spawn borers due to all other events hitting their max occurrences